runelite-api/src/main/java/net/runelite/api/Varbits.java

```diff
@@ -683,6 +683,13 @@
 	 */
 	public static final int TELEBLOCK = 4163;
 
+	/**
+	 * If the player has Menaphite remedy effect active.
+	 * This will go down by 1 every 25 ticks (15 seconds) and the player&#39;s combat stats will be restored by 6 + 16%.
```

Comment:
This apostrophe doesn't need escaping.
```suggestion
	 * This will go down by 1 every 25 ticks (15 seconds) and the player's combat stats will be restored by 6 + 16%.
```